.travis.yml

```diff
@@ -224,6 +224,23 @@ language: php
       jobTests+="${jobTest}\n"
       echo "${jobTest}"
       echo "----------------------"
+  - &build_test_validation |
+      failTest=false
+      echo "-------------"
+      echo "Run Validation Tests"
+      echo "-------------"
+      docker exec -i $(docker ps | grep _openemr | cut -f 1 -d " ") sh -c "cd ${OPENEMR_DIR}; php ${OPENEMR_DIR}/vendor/bin/phpunit --testsuite validations --testdox" || failTest=true
+      if $failTest; then
+        failJob=true
+        mes="FAILED"
+      else
+        mes="PASSED"
+      fi
+      echo "----------------------"
+      jobTest="${mes} - Run Validation Tests"
+      jobTests+="${jobTest}\n"
+      echo "${jobTest}"
+      echo "----------------------"      
```

Comment:
Is all the above supposed to be Validations or Validators?

Reply:
Good catch!

Reply:
Updated to Validators!